app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -119,13 +145,227 @@ class DeveloperOptionsActivityTest {
     Truth.assertThat(title).isEqualTo(context.getString(R.string.developer_options_title))
   }
 
+  @Test
```

Comment:
I think we can add a few more tests to check the hints and solutions' switch status. These can be similar to the ones we do for wifi switch in administrator controls. Additionally in the future (when the corresponding activities are made), we'll need to add tests related to opening of appropriate activity on option selection in this test file as well.

Reply:
I have added the test for the switch. PTAL. Thanks.

Reply:
Aah, I think there was one more test involving config change.

Reply:
Yes. But that test is feasible only after implementing the logic for showing hints and solution. We can add that test when we implement this or we can add the test now and mark it as **Ignore**.

Reply:
Oh okay, maybe we can put it in place at the time of logic implementation.